### ISSUE TITLE
sd_amplifier set to 1.3 for de-noising?

### ISSUE THREAD

**Issue by RegnerM2015** [Swarbricklab-code/BrCa_cell_atlas#11]
I see that the following code was used to run InferCNV:

```
infercnv::run(
        initial_infercnv_object,
        num_threads=numcores-1,
        out_dir=out_dir,
        cutoff=0.1,
        window_length=101,
        max_centered_threshold=3,
        cluster_by_groups=F,
        plot_steps=F,
        denoise=T,
        sd_amplifier=1.3,
        analysis_mode = "samples"
      )
```

The default value for `sd_amplifier` is 1.5. I was curious about your decision to adjust this de-noising parameter? I was hoping you/you all would be able to share your insights that could help me plan my future analyses and pipelines. Thank you!

**Comment by dlroden** [comment]
Hi there, sorry the slow response. Regarding the denoising parameter, we found that 1.3 performed slightly better in terms of accuracy when we tested it on simulated cancer datasets, and still did a good job at denoising when tested it on a normal breast tissue dataset. We expect that this change in the de-noising parameter will not make a large difference to the results and could differ depending on different datasets, but was found to be optimal in our case. This benchmarking is currently unpublished but we are working on getting it written up. 

I hope this information helps with setting up your pipelines and future studies. Thanks
